tests/structuredheadings/schemediscovery.js

```diff
@@ -158,7 +158,38 @@
       );
       wait();
 
-    }
+    },
 
+    "autodetect will not destroy existing list customizations in non-autonumber doc": function() {
```

Comment:
The root cause wasn't caused by what this module tests, (it was a far simpler configuration issue) but it's probably good to test that the scheme detection magic doesn't remove your styles, so I'm keeping it here.